pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -245,8 +244,49 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (desiredPodCount
 	return desiredPodCount, int32(excessBCF), true
 }
 
+func clampInclusiveRange(valueToClamp float64, lowerBound float64, upperBound float64) float64 {
```

Comment:
```suggestion
func clampInclusiveRange(valueToClamp, lowerBound, upperBound float64) float64 {
```

Reply:
Updated in 94457ef.

I always forget this bit of syntax.